Userland/Libraries/LibTorrent/Bencode/Bencode.cpp

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2021, Cesar Torres <shortanemoia@protonmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Bencode.h"
+#include <AK/Format.h>
+#include <AK/NonnullOwnPtrVector.h>
+#include <AK/Optional.h>
+#include <AK/OwnPtr.h>
+#include <AK/String.h>
+
+namespace Bencode {
```

Comment:
```suggestion
namespace Torrent::Bencode
```
If it's in a subdirectory of LibTorrent, it should be under the `Torrent` namespace.